streams/src/test/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetricsRecorderTest.java

```diff
@@ -173,6 +173,22 @@ public void shouldThrowIfMetricRecorderIsReInitialisedWithDifferentTask() {
         );
     }
 
+    @Test
+    public void shouldThrowIfMetricRecorderIsInitialisedWithNullMetrics() {
+        assertThrows(
+            IllegalStateException.class,
```

Comment:
This should be `NullPointerException`

Build is actually failing...

Reply:
forgot to actually run the new tests I guess...whoops 😅